.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Ask a question
+    url: https://community.planet.com/planet-s-community-forum-3
```

Comment:
We probably should turn off our 'discussions' in github if we're pointing people at Planet community (which I do think is probably the right thing).

Reply:
Agreed. Is pointing to planet community the right thing to do? Will we have someone from this team monitoring the forum or will a monitor connect with someone from our team if an SDK question comes up?

Reply:
Yeah, I think we should be able to monitor, or at least be notified if questions about the SDK come in.